Shared/Samples/Analyze network with subnetwork trace/AnalyzeNetworkWithSubnetworkTraceView.swift

```diff
@@ -0,0 +1,314 @@
+                        }
+                    }
+                }
+                .disabled(selectedAttribute == nil && selectedComparison == nil)
+            } else {
+                HStack {
+                    Text("Value")
+                    TextField("Comparison value", value: $inputValue, format: .number, prompt: Text("Value"))
+                        .multilineTextAlignment(.trailing)
+                        .lineLimit(1)
+                }
+                .onChange(of: inputValue) { value in
+                    selectedValue = value
+                }
+                .disabled(selectedAttribute == nil && selectedComparison == nil)
+            }
+        }
+        .navigationTitle("Add Condition")
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .confirmationAction) {
+                Button("Done") {
+                    isConditionMenuPresented = false
+                    guard let attribute = selectedAttribute,
+                          let comparison = selectedComparison,
+                          let value = selectedValue else {
+                        // show error
+                        return
+                    }
+                    do {
+                        try model.addConditionalExpression(
+                            attribute: attribute,
+                            comparison: comparison,
+                            value: value
+                        )
+                        selectedAttribute = nil
+                        selectedComparison = nil
+                        selectedValue = nil
+                        inputValue = nil
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .disabled(
+                    selectedAttribute == nil ||
+                    selectedComparison == nil ||
+                    selectedValue == nil
+                )
+            }
+            ToolbarItem(placement: .cancellationAction) {
+                Button("Cancel") {
+                    isConditionMenuPresented = false
+                    selectedAttribute = nil
+                    selectedComparison = nil
+                    selectedValue = nil
+                    inputValue = nil
+                }
+            }
+        }
+        .navigationViewStyle(.stack)
+    }
+    
+    @ViewBuilder var attributesView: some View {
+        List(model.possibleAttributes, id: \.name) { attribute in
+            HStack {
+                Text(attribute.name)
+                if attribute === selectedAttribute {
+                    Spacer()
+                    Image(systemName: "checkmark")
+                        .foregroundColor(.accentColor)
+                }
+            }
+            .contentShape(Rectangle())
+            .onTapGesture {
+                selectedAttribute = attribute
+            }
+            .navigationTitle("Attributes")
+        }
```

Comment:
The title should go on the list itself.

```suggestion
        }
        .navigationTitle("Attributes")
```